libs/auth/src/openssl.test.ts

```diff
@@ -493,37 +501,147 @@ test.each<{
     expect(spawn).toHaveBeenCalledTimes(1);
     if (isSudoExpected) {
       expect(spawn).toHaveBeenNthCalledWith(1, 'sudo', [
-        expect.stringContaining('/src/create-cert'),
+        expect.stringContaining('/src/intermediate-scripts/create-cert'),
         expect.any(String),
       ]);
     } else {
       expect(spawn).toHaveBeenNthCalledWith(
         1,
-        expect.stringContaining('/src/create-cert'),
+        expect.stringContaining('/src/intermediate-scripts/create-cert'),
         [expect.any(String)]
       );
     }
   }
 );
 
-test('createCert - process exits with an error code', async () => {
+//
+// Message signing functions
+//
+
+test.each<SignMessageInputExcludingMessage>([
+  { signingPrivateKey: { source: 'file', path: '/path/to/private-key.pem' } },
+  { signingPrivateKey: { source: 'tpm' } },
+])(
+  'parseSignMessageInputExcludingMessage success',
+  (signMessageInputExcludingMessage) => {
+    expect(
+      parseSignMessageInputExcludingMessage(
+        JSON.stringify(signMessageInputExcludingMessage)
+      )
+    ).toEqual(signMessageInputExcludingMessage);
+  }
+);
+
+test.each<string>([
+  '',
+  JSON.stringify({}),
+  JSON.stringify({ signingPrivateKey: { source: 'file', oops: 'oops' } }),
+])(
+  'parseSignMessageInputExcludingMessage error',
+  (signMessageInputExcludingMessage) => {
+    expect(() =>
+      parseSignMessageInputExcludingMessage(signMessageInputExcludingMessage)
+    ).toThrow();
+  }
+);
+
+test.each<{
+  description: string;
+  signingPrivateKey: SignMessageInput['signingPrivateKey'];
+  expectedOpensslSignatureRequestParams: string[];
+}>([
+  {
+    description: 'signing with private key file',
+    signingPrivateKey: { source: 'file', path: '/path/to/private-key.pem' },
+    expectedOpensslSignatureRequestParams: [
+      'dgst',
+      '-sha256',
+      '-sign',
+      '/path/to/private-key.pem',
+    ],
+  },
+  {
+    description: 'signing with TPM private key',
+    signingPrivateKey: { source: 'tpm' },
+    expectedOpensslSignatureRequestParams: [
+      'dgst',
+      '-sha256',
+      '-sign',
+      TPM_KEY_ID,
+      '-keyform',
+      'engine',
+      '-engine',
+      OPENSSL_TPM_ENGINE_NAME,
+      '-passin',
+      `pass:${TPM_KEY_PASSWORD}`,
+    ],
+  },
+])(
+  'signMessageHelper - $description',
+  async ({ signingPrivateKey, expectedOpensslSignatureRequestParams }) => {
+    setTimeout(() => {
+      // process.stdin.destroy();
```

Comment:
remove?

Reply:
🙈 Thank you for catching! https://github.com/votingworks/vxsuite/pull/3518/commits/f0b5c7a6c083e1c13f5d82a3d5e080e41b99607f